src/operator/tensor/dot-inl.h

```diff
@@ -573,6 +577,18 @@ inline void DotCsrDnsDnsImpl(const OpContext& ctx,
   });
 }
 
+
+struct MarkCsrColKernel {
```

Comment:
Can we reuse this kernel and give it a better name?
https://github.com/apache/incubator-mxnet/blob/master/src/operator/tensor/util/tensor_util-inl.h#L37